### DISCUSSION TITLE
这个不能设置从什么时候运行吗？

### DISCUSSION THREAD

**Discussion by longwangjiang** [luolongfei/freenom#241] [General]
这个不能设置从什么时候运行吗？比如我的域名续期日期是2024年5月22号，我想从2024年5月10日开始续期，这样就不会过多的访问freenom了，这个能设置吗？还有就是多个号不同日期续期可以分别定时运行续费操作吗？谢谢大佬解答运行！

**Comment by luolongfei** [comment]
有个 RUN_AT 变量支持 corn 表达式，具体看文档
